main.py

```diff
@@ -21,7 +46,7 @@ def create_user(user: UserModel, status_code=201):
     """
 
     s = repo.create_user(user.dict())
-    s['token'] = JWTAuth.generate_jwt_token(s)
+    #s['token'] = JWTAuth.generate_jwt_token(s)

```

Comment:
to w przyszłości trzeba będzie uwarunkować jakimś ENV variable typu "useMockAuthentication"